lib/Facebook.js

```diff
@@ -815,9 +815,46 @@ function Facebookbot(configuration) {
         });
     };
 
+    var insights_api = {
+        get_insights: function (metrics, since, until, cb) {
+            var uri = 'https://' + api_host + '/v2.8/me/insights?access_token=' + configuration.access_token;
 
+            if (metrics) {
+                if (typeof(metrics) == 'string') {
+                    uri += '&metric=' + metrics;
+                } else {
+                    uri += '&metric=' + metrics.join(',');
+                }
+            }
 
+            if (since) {
+                uri += '&since=' + since;
+            }
 
+            if (until) {
+                uri += '&until=' + until;
+            }
+
+            request({
+                method: 'GET',
+                json: true,
+                uri: uri
+            }, function (err, res, body) {
+                if (err) {
+                    facebook_botkit.log('Could call Facebook Insights API');
```

Comment:
I think that this log message is wrong.
Also logging err.message helps with debugging.

Reply:
Nice catch 👍